src/Stripe.Tests.XUnit/connect/when_charging_from_managed_account.cs

```diff
@@ -38,7 +40,14 @@ public charging_from_another_account()
             var appFeeService = new StripeApplicationFeeService(Cache.ApiKey);
             appFeeService.ExpandApplication = true;
 
-            _appFee = appFeeService.Get(charge.ApplicationFeeId);
+            _appFee = appFeeService.Get(_charge.ApplicationFeeId);
+        }
+
+        [Fact]
+        public void it_should_have_connected_app_on_charge()
+        {
+            _charge.ApplicationId.Should().StartWith("ca_");
+            _charge.Application.Should().NotBeNull();
```

Comment:
Is test order execution guaranteed in XUnit? I'd expect to see the charge initialization in a setup method or a new charge explicitly created for this test.

Reply:
agreed, usually this is done in a fixture first no?

Reply:
I'm not entirely sure what you two meant, or if you're even talking about the same thing, but:
- no, XUnit does not guarantee test order by default
- using a fixture allows for context sharing between tests (cf. https://xunit.github.io/docs/shared-context.html). I updated the test to use a fixture to avoid creating multiple resources needlessly and make the tests a bit faster